### ISSUE TITLE
Request: build python whl  package 

### ISSUE THREAD

**Issue by sweetcard** [iamlemec/bert.cpp#7]
It could be installed by pip and can be used in other projects.
Thanks.

**Comment by iamlemec** [comment]
Definitely. Aiming to make this an analog of `llama-cpp-python` but for embeddings. Should have a release ready in a few days.

**Comment by sweetcard** [comment]
That's cool. Thank u for your amazing work.
